src/components/layout/GenericLayout/Footer/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import styled from 'styled-components'
+import { depositApi } from 'api'
+
+// Components
+import { BlockExplorerLink } from 'components/common/BlockExplorerLink'
+
+// Hooks
+import { useWalletConnection } from 'hooks/useWalletConnection'
+
+// Config
+import { footerConfig } from '../Footer/config'
+
+const FooterStyled = styled.footer`
+  position: fixed;
+  font-size: 1.1rem;
+  bottom: 0;
+  height: 3rem;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 1rem;
+  border-top: 0.1rem solid var(--color-border);
+  flex: 1 1 auto;
+  background: var(--color-gradient-1);
+  width: 100%;
+  color: var(--color-text-secondary2);
+  justify-content: space-between;
+`
+
+const VerifiedButton = styled(BlockExplorerLink)`
+  margin: 0 auto 0 0.6rem;
+`
+
+const VersionsWrapper = styled.div`
+  display: flex;
+  margin: 0 0 0 auto;
+
+  > a {
+    text-decoration: none;
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
```

Comment:
```suggestion
  > a {
    &,
    &:focus,
    &:hover {
      text-decoration: underline;
    }
  }
```

Reply:
Care to elaborate? Never seen it in use like this :)

Reply:
<img width="263" alt="Screen Shot 2020-11-23 at 10 05 26" src="https://user-images.githubusercontent.com/31534717/99944585-6b52a780-2d73-11eb-8e3a-27740a679c54.png">

Is what I get. So in this case we merged all styles (<a> element + the focus/hover pseudo). But I intended it to be `text-decoration: none` by default and only on hover have the text-decoration.

Reply:
`&` duplicates current selector, [try here](https://stylus-lang.com/try.html#?code=%3E%20a%20%7B%0A%20%20%20%20%26%2C%0A%20%20%20%20%26%3Afocus%2C%0A%20%20%20%20%26%3Ahover%20%7B%0A%20%20%20%20%20%20text-decoration%3A%20underline%3B%0A%20%20%20%20%7D%0A%20%20%7D)

Reply:
ok, disregard me, I misread tex-decoration values as the same, shame on me :disappointed:

Reply:
Still good to explore options! Thanks for checking :)